Expensify/App
# add regex for US account number

### Details
Added regex for US account number


### Fixed Issues

$ https://github.com/Expensify/App/issues/7056

### Tests



### QA Steps

1. Open app
2. Navigate to settings > Workspace > Add bank account
3. Proceed to the manual flow
4. Add alphabets in account number
- [x] Verify that account number shows error for alphanumeric characters.
- [x] Verify that account number works for numbers of length 4-17

### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/83179295/148772918-15e7edea-e261-4bc2-b92a-2bdef153737a.mov



#### Mobile Web


#### Desktop


#### iOS


#### Android
